site/app/templates/grading/electronic/SubmissionPanel.twig

```diff
@@ -49,37 +49,35 @@
                 {% endif %}
             </div>
             {# /Files #}
-            <script>
-                // $(document).ready(function(){
-                //     $(".openAllDiv").on('click', function(){
-                //         if($(this).attr('id') == 'results' || $(this).attr('id') == 'submissions' || $(this).attr('id') =='checkout'){
-                //             var elem = $('[data-linked-type="' + $(this).attr('id') + '"]');
-                //             if(elem.data('clicked-state') == "wasntClicked"){
-                //                 updateValue(elem, "Open", "Close");
-                //             }
-                //         }
-                //     });
-                // });
-            </script>
         </div>
-        <div id="file_view" style="display: none; position: absolute; float: right; right: -100%; width:100%; height: 100%">
-            <strong id="save_status" style="position: sticky; top: 27px; margin: 10px; float:right; z-index: 30; display: none">No changes</strong>
-            <span class="grading_label" style="padding-bottom: 0px">Submissions and Results Browser ></span>
-            <span class="grading_label" style="padding-top: 5px; margin-left: -15px" id="grading_file_name"></span>
-            <a onclick='collapseFile()' aria-label="Collapse File" class="key_to_click" tabindex="0"><i class="fas fa-reply" style="font-size:24px" title="Back"></i></a>
-            {% include "grading/electronic/PDFAnnotationBar.twig" %}
-            <div id="file_content" style="overflow: auto; height:90%;position: absolute;width: 100%;"></div>
+        <div id="file_view" class="hide">
+            <div class="sticky-file-info">
+                <div class="flex-row">
+                    <span>
+                        <span class="grading_label">Submissions and Results Browser ></span>
+                        <span class="grading_label" id="grading_file_name"></span>
```

Comment:
For writing new CSS/HTML we should use dashes isntead of underscores for HTML IDs

https://submitty.org/developer/coding_style_guide/css

Reply:
Actually I didn't change the existing classes and ids as sometimes it breaks the tests...but done now :)